server.js

```diff
@@ -11,6 +12,7 @@ app.get('/api/countries/:name', async (req, res) => {
     // Extract the country name from the URL path parameter
     const countryName = req.params.name;
     const url = `https://restcountries.com/v3.1/name/${countryName}?fullText=true`;
+    country = countryName;
 
```

Comment:
Good idea naming it "country". It makes it super clear, especially when we reference it in the template literal.